src/icon-tab-bar.scss

```diff
@@ -252,6 +261,11 @@ $fd-icon-tab-bar-semantic-values: (
     list-style: none;
     box-shadow: var(--sapContent_HeaderShadow);
     background: var(--sapObjectHeader_Background);
+    position: relative;
+
+    &--left-offset {
+      padding-left: 3.5rem;
```

Comment:
I am not seeing this class used anywhere - and typically any sort of padding-left will need its corresponding padding-right for RTL mode. See the new `fd-set-padding-left` mixin

Reply:
+